parts/footer.html

```diff
@@ -1,6 +1,6 @@
 <!-- wp:group {"layout":{"inherit":true}} -->
 <div class="wp-block-group">
-	<!-- wp:group {"style":{"spacing":{"padding":{"top":"80px","bottom":"30px"}}}} -->
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"5rem","bottom":"5rem"}},"width":{"1000px"}}} -->
```

Comment:
I wonder if we should use fixed units here so that the vertical padding isn't so unpredictable. Generally speaking, I would follow what the base theme does (e.g. use `px`). Also, I'm not sure we should set a width for footer. We can leave the footer details for a future card.

Reply:
Ok, I removed footer changes, I guess footer will be changed a lot 
2c77e935d71a1e1137bcda04a2876bcfe6f90ddb